vpr/src/place/net_cost_handler.cpp

```diff
@@ -1478,7 +1489,7 @@ double NetCostHandler::get_net_cube_bb_cost_(ClusterNetId net_id, bool use_ts) {
     ncost = (bb.xmax - bb.xmin + 1) * crossing * chanx_place_cost_fac_[bb.ymax][bb.ymin - 1];
     ncost += (bb.ymax - bb.ymin + 1) * crossing * chany_place_cost_fac_[bb.xmax][bb.xmin - 1];
     if (is_multi_layer) {
-        ncost += (bb.layer_max - bb.layer_min) * crossing * chanz_place_cost_fac_[bb.xmax][bb.ymax][bb.xmin][bb.ymin];
+        ncost += (bb.layer_max - bb.layer_min) * crossing * get_chanz_cost_factor(bb);
```

Comment:
Can't we multiply `crossing` once at the end? It would save us two doulbe precision multiplications